src/main/java/sparksoniq/semantics/types/SequenceType.java

```diff
@@ -74,6 +87,12 @@ public boolean equals(Object o) {
         if (!(o instanceof SequenceType))
             return false;
         SequenceType sequenceType = (SequenceType) o;
```

Comment:
naming this variable 'other' could be nice for readability

Reply:
Done